plugins/spark/entrypoint.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
```

Comment:
dont required right?

Reply:
Yeah, this was temporary but should be removed once we have the flytekit fix checked in